python/cudf/comm/serialize.py

```diff
@@ -1,8 +1,8 @@
-# A flag to allow dask_gdf to detect and warn if
-# IPC serialization is unavailable
-CUSTOM_SERIALIZATION_AVAILABLE = False
+import functools
 
+from distributed.protocol.cuda import cuda_serialize, cuda_deserialize
+from distributed.protocol import serialize, deserialize
```

Comment:
Would it be possible to move these imports down into the `register_distributed_serializer` function to make them optional?